src/verifier/request_signer.rs

```diff
@@ -42,12 +44,22 @@ impl P256Signer {
         Ok(Self { key, jwk })
     }
 
+    pub fn from_pkcs8_pem(s: &str) -> Result<Self> {
```

Comment:
nit: I don't think this should be added to the crate. It means we make the pkcs8 feature of p256 mandatory, when a dependent might just want to use DER